content/en/docs/ip-cert-privacy.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Privacy Considerations for IP Certs
+slug: ip-cert-privacy
+lastmod: 2025-10-22
+show_lastmod: true
+---
+
+# Privacy Considerations for Home Use of IP Address Certificates
+
+Since [July 2025](https://letsencrypt.org/2025/07/01/issuing-our-first-ip-address-certificate), Let’s Encrypt has been able to issue certificates for IP addresses, in addition to its traditional certificates covering domain names.
```

Comment:
Avoid using "smart quotes", here and throughout.
```suggestion
Since [July 2025](https://letsencrypt.org/2025/07/01/issuing-our-first-ip-address-certificate), Let's Encrypt has been able to issue certificates for IP addresses, in addition to its traditional certificates covering domain names.
```